falcon/request.py

```diff
@@ -256,6 +257,9 @@ def __init__(self, env, options=None):
         self.stream = env['wsgi.input']
         self.method = env['REQUEST_METHOD']
 
+        if self.stream.__class__.__name__ == InputWrapper.__name__:
```

Comment:
Okay, I had to delete the Cython-compiled files before my debug `print()`s started showing. It turns out, yes, adding `or isinstance(self.stream, InputWrapper):` to to L306 is enough to get testing working. So much for this PR :cry: